docs/guides/controller-autoconfiguration.md

```diff
@@ -36,34 +37,53 @@ This allows:
 
-
 1. Unplug all the other joypads
-2. Use `Settings` -> `Input` -> `User 1 Binds` -> `User 1 Bind All`. Make sure that your mapping is perfect by testing every button in the menu and in some games.
-3. Use `Settings` -> `Inputs` -> `User 1 Binds` -> `User 1 Save Autoconfig`
-4. The new profile will be saved to your disk.
-5. Use `Settings` -> `Inputs` -> `User 1 Binds` -> `User 1 Bind Default All` to clear manual bindings and rely on the new profile
-6. Unplug your joypad an re-plug it. See if it is auto configured.
+2. For Android, run the [Input descriptors](#android) step first.
+3. Use `Settings` -> `Input` -> `RetroPad Binds` -> `Port 1 Controls` -> `Set All Controls`. Make sure that your mapping is perfect by testing every button in the menu and in some games.
+4. If applicable, then also set the menu button binding in `Settings` -> `Input` -> `Hotkeys` -> `Menu Toggle`
+5. Use `Settings` -> `Input` -> `RetroPad Binds` -> `Port 1 Controls` -> `Save Controller Profile`
+6. The new profile file will be saved to your disk: [Controller profile directory]/[Controller driver]/[Device index].cfg
+7. Use `Settings` -> `Inputs` -> `Port 1 Controls` -> `Reset to Default Controls` to clear manual bindings and rely on the new profile
+8. Unplug your joypad an re-plug it. See if it is auto configured.
 
 
 !!! Warning "Clear manual bindings"
-    It is important to to skip the step of clearing manual bindings after using the `User 1 Save Autoconfig` command. In order to avoid issues using your profile in the future, remember to go to `Settings` -> `Inputs` -> `User 1 Binds` -> `User 1 Bind Default All` to reset the manual settings before completing this process.
+    It is important to to skip the step of clearing manual bindings after using the `Save Controller Profile` command. In order to avoid issues using your profile in the future, remember to go to `Settings` -> `Input` -> `RetroPad Binds` -> `Port 1 Controls` -> `Reset to Default Controls` to reset the manual settings _before_ completing this process.
 
 If you are happy with your profile, you can submit it to RetroArch so that other users benefit:
 
-1. Edit the autoconfig file for your joypad manually to include the input descriptors (please see the **Input Descriptors** section below)
-2. [Submit your profile to our joypad profile repository](https://github.com/libretro/retroarch-joypad-autoconfig).
+1. Edit the autoconfig file for your joypad manually to include the input descriptors (please see the [Input descriptors](#input-descriptors) section below)
+3. [Submit your profile to our joypad profile repository](https://github.com/libretro/retroarch-joypad-autoconfig).
+
+### Android
+
+Android phones that are not rooted you need to run these steps in order to backup your profile files or to [Submit your profile to our joypad profile repository](https://github.com/libretro/retroarch-joypad-autoconfig):
+- Create `/storage/emulated/0/RetroArch/autoconfig/`
```

Comment:
I believe RetroArch on Android creates this folder for us automatically. How did you install Android on your device? You should use the distributed `.apk`.

Reply:
> I believe RetroArch on Android creates this folder for us automatically.

* /storage/emulated/0/RetroArch/autoconfig is created by default
* /storage/emulated/0/RetroArch/autoconfig/ is not created by default. By the way I just changed it to /storage/emulated/0/RetroArch/autoconfig/android in the doc, otherwise the autoconfig files will be stored in /storage/emulated/0/RetroArch/autoconfig/

> How did you install Android on your device? You should use the distributed `.apk`.

Don't worry. I stopped using the F-Droid release (unless stated) long a few weeks ago.
I'm using https://buildbot.libretro.com/stable/1.17.0/android/RetroArch.apk